tests/gdimageconvolution/bug00369.c

```diff
@@ -0,0 +1,35 @@
+/**
+ * Test Issue #369 for gdImageConvolution()
+ */
+
+#include "gd.h"
+#include "gdtest.h"
+
+int main()
+{
+	gdImagePtr im;
+	int pxl;
+	int error = 0;
+	float matrix[3][3] = {
+		{1, 0, 1},
+		{0, 5, 0},
+		{1, 0, 0}
+	};
+
+	im = gdImageCreateTrueColor(40, 40);
+	gdImageAlphaBlending(im, gdEffectReplace);
+	gdImageFilledRectangle(im, 0, 0, 39, 39, 0x7FFFFFF);
+	gdImageFilledEllipse(im, 19, 19, 20, 20, 0x00FF00);
+	gdImageSaveAlpha(im, 1);
```

Comment:
This call is superfluous if the image is not saved (see http://libgd.github.io/manuals/2.2.5/files/gd-c.html#gdImageSaveAlpha).

Reply:
your are right. i will delete it.